opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -80,7 +80,12 @@ def __init__(
         self, context: "SpanContext", attributes: types.Attributes = None
     ) -> None:
         self._context = context
-        self._attributes = attributes
+        if attributes is None:
+            self._attributes = (
+                {}
+            )  # type: typing.Dict[str, types.AttributeValue]
```

Comment:
```suggestion
            )  # type: types.Attributes
```

Any reason not to use our type alias here too?

Reply:
No reason, makes sense to use the alias. I've updated the code.